make/debian-trust-package.mk

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+debian_package_layer := $(bin_dir)/scratch/debian-trust-package
+debian_package_json := $(debian_package_layer)/debian-package/cert-manager-package-debian.json
+
+$(debian_package_layer)/debian-package $(bin_dir)/bin:
+	mkdir -p $@
+
+$(bin_dir)/bin/validate-trust-package: | $(bin_dir)/bin
+	go build -o $@ ./cmd/validate-trust-package
```

Comment:
suggestion: This target has dependencies which aren't listed here which will prevent it being rebuilt correctly.

Needs at least `cmd/validate-trust-package/main.go` and `pkg/fspkg/package.go`

Reply:
Fixed